java/src/jmri/jmrix/AbstractThrottle.java

```diff
@@ -100,17 +97,23 @@ public AbstractThrottle(SystemConnectionMemo memo) {
      * @param memo System Connection this throttle is on
      * @param totalFunctions total number of functions available, including 0
      */
-    public AbstractThrottle(SystemConnectionMemo memo, int totalFunctions) {
+    public AbstractThrottle(@Nonnull SystemConnectionMemo memo, int totalFunctions) {
         active = true;
         adapterMemo = memo;
         FUNCTION_BOOLEAN_ARRAY = new boolean[totalFunctions];
         FUNCTION_MOMENTARY_BOOLEAN_ARRAY = new boolean[totalFunctions];
     }
 
     /**
-     * System Connection this throttle is on
+     * Get the System Connection this throttle is on.
+     * @return non-null system connection.
      */
-    protected SystemConnectionMemo adapterMemo;
+    @Nonnull
+    protected SystemConnectionMemo getMemo() {
+        return adapterMemo;
+    }
+
+    protected final SystemConnectionMemo adapterMemo;
```

Comment:
No need to change, but since you have added the method, it might be good to make this variable private.